extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/OtlpRecorder.java

```diff
@@ -31,6 +31,10 @@ public void installBatchSpanProcessorForOtlp(OtlpExporterConfig.OtlpExporterRunt
                     Map<String, String> headers = OpenTelemetryUtil.convertKeyValueListToMap(runtimeConfig.headers.get());
                     headers.forEach(otlpGrpcSpanExporterBuilder::addHeader);
                 }
+                if (runtimeConfig.enableCompression) {
+                    // Currently, the only supported compression method is "gzip".
+                    otlpGrpcSpanExporterBuilder.setCompression("gzip");
```

Comment:
Is there an expectation for this to have alternatives in the future or no?

Reply:
According to the spec, gzip is the only supported, but language SDKs can provide other compressions that should be documented. Don't think that we will have another compression supported soon. What we can do be able to accommodate this change is leave as is, and in the feature add a parameter `compressionType` or `compressionCodec`. Or we can add it now and leave `gzip` as the default, we can leave this as a string and not use an enum, in this case if a codec is added to the Java SDK we don't need to change anything, or we can enforce using an enum, but whenever a new codec is added we would have to also update Quarkus to add the new codec.

Reply:
Used `compression` as an option string and it points to the otel spec, so whenever there is a new option we will be able to support right away.